### DISCUSSION TITLE
Bug? Music added to Submariner Library as linked file won't play after quit / relaunch.

### DISCUSSION THREAD

**Discussion by raxelbrof** [SubmarinerApp/Submariner#196] [Q&A] [answered]
Hi,

So trying out Submariner, current version from Mac App Store. I add music as linked file that's stored on an SMB shared volume hosted by another Mac (No subsonic server yet). It will play fine after it's added, but if I quit Submariner, and relaunch, the files won't play. I can add new tracks in this condition, and those files will play fine, but again, quit, relaunch, and those newly added files won't play.

The Play / Pause control changes to the pause control, but the playhead doesn't progress from 00:00.

I assume this isn't "normal" behaviour?

**Comment by NattyNarwhal** [comment, marked as answer]
Looks like a sandbox thing; we need to store a bookmark to be access to access a directory after relaunching.
